cuenca/resources/card_transactions.py

```diff
@@ -0,0 +1,35 @@
+import datetime as dt
+from dataclasses import dataclass
+from typing import ClassVar, List, cast
+
+from cuenca_validations.types import (
+    CardErrorType,
+    CardNetwork,
+    CardTransactionType,
+    CardType,
+)
+
+from .base import Transaction
+from .resources import retrieve_uri
+
+
+@dataclass
+class CardTransaction(Transaction):
+    _resource: ClassVar = 'card_transactions'
+
+    type: CardTransactionType
+    network: CardNetwork
+    related_card_transaction_uris: List[str]
+    card_last4: str
+    card_type: CardType
+    merchant: str
+    metadata: dict
```

Comment:
What does this have?

Reply:
Here we will return CardTransaction fields necessaries when any clarification. Fields like: `pos_capabilty`, `prosa_transaction_id`, `retrieval_reference `, `transaction_reference `, `track_data_method `, `authorizer_number `, `currency_code `,

Reply:
Why is any of that necessary? Is it used by the app?

Reply:
Not by the app, but for Omni and support team is necessary. We prefer to have `metadata` fields available in Cuenca Python than querying db for more info.